Configurations/50-win-onecore.conf

```diff
@@ -82,7 +82,35 @@ my %targets = (
         ex_libs         => "onecore.lib",
         multilib        => "-arm64",
     },
-
+    "VC-WIN64-CLANGASM-ARM" => {
+        inherit_from    => [ "VC-noCE-common" ],
+        defines         => add("_ARM_WINAPI_PARTITION_DESKTOP_SDK_AVAILABLE",
+                               "OPENSSL_SYS_WIN_CORE"),
```

Comment:
Please add `"__ASSEMBLER__"` here

Reply:
@levitte Thanks a lot for the feedback. Adding it directly here will define it for all makefile targets. If I understand the use of the flag correctly it is just enabled to tell the compiler we're preprocessing an assembler file https://gcc.gnu.org/onlinedocs/gcc-5.1.0/cpp/Standard-Predefined-Macros.html which is not always true. This is the reason for the hack on the template file.